Jellyfin.Data/Jellyfin.Data.csproj

```diff
@@ -1,12 +1,26 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
What's the reason to keep both of these? I'm not really familiar with the targeting setup for the solution, so this is more of a question

Reply:
So @Bond-009 said that we need this project to target netstandard 2.0, and the rest of the project targets 2.1, so I'm taking the same approach as `MediaBrowse.Model` and targeting both

Reply:
Ah okay, yeah makes sense to match what is in `MediaBrowser.Model`